src/System.Windows.Forms/src/System/Windows/Forms/PropertyGridInternal/GridToolTip.cs

```diff
@@ -189,37 +153,33 @@ private void SetupToolTip(Control c)
                                   NativeMethods.SWP_NOMOVE | NativeMethods.SWP_NOSIZE |
                                   NativeMethods.SWP_NOACTIVATE);
 
-                if (0 == (int)UnsafeNativeMethods.SendMessage(new HandleRef(this, Handle), NativeMethods.TTM_ADDTOOL, 0, GetTOOLINFO(c)))
+                if (GetTOOLINFO(c).SendMessage(this, WindowMessages.TTM_ADDTOOLW) == IntPtr.Zero)
```

Comment:
A bit more verbose, but slightly more readable
```suggestion
                var toolInfo = GetTOOLINFO(c);
                var result = toolInfo.SendMessage(this, WindowMessages.TTM_ADDTOOLW);
                if (result == IntPtr.Zero)
```